scripts/rca-demo/README.md

```diff
@@ -0,0 +1,9 @@
+# RCA KubeCon Demo Scripts
+
+Docs TBA for running scripts in this folder. 
```

Comment:
Can you just copy over the "Fast Track" stuff from the README?

Reply:
Oh, for some reason I had an outdated version of the main README so I couldn't find these sections to copy. I've found them now and moved things over.